photocollage/gtkgui.py

```diff
@@ -20,6 +20,10 @@
 import cairo
 import copy
 import gettext
+import gi
+
+gi.require_version('Gtk', '3.0')
```

Comment:
Hi, unfortunately this must be done before the import. The warning is:
  Use gi.require_version('Gtk', '3.0') before import to ensure that the right version gets loaded.

I tried to move it but it didn't help. It seems that the warning is generated at the moment the import line is processed.

I can put following:
import gi
gi.require_version('Gtk', '3.0')
from gi.repository import Gtk, Gdk, GObject

before "import cairo" or drop the commit